src/ashell/ide-cmd.c

```diff
@@ -0,0 +1,743 @@
+// Copyright (c) 2017-2018, Intel Corporation.
+
+// C includes
+#include <ctype.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <string.h>
+#include <malloc.h>
+
+// Zephyr includes
+#include <atomic.h>
+#include <misc/printk.h>
+#include <misc/reboot.h>
+#include <zephyr/types.h>
+
+#ifdef CONFIG_BOARD_ARDUINO_101
+  #include <flash.h>
+#endif
+
+// JerryScript includes
+#include "jerry-code.h"
+
+// ZJS includes
+#include "../zjs_util.h"
```

Comment:
this is not right,  should just set the include paths when you build instead of doing ../